Documentation/ceph-csi-drivers.md

```diff
@@ -51,15 +48,21 @@ running in. When Ceph CSI is deployed by Rook, the operator will automatically
 maintain a config map whose contents will match this key. By default this is
 "rook-ceph".
 
+If you've deployed the Rook operator in a namespace other than "rook-ceph",
```

Comment:
This is true for any usage of the csi driver. How about adding a new heading previous to the snapshot section for configuration of the csi driver when in a non-default namespace?